Tasks/Doroshevich/Task2/Task2.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+
+choice = input(
+    "Введите '1'- Для отображения текущего времени\nВведите '2'- Если хотите ввести время с консоли\n")
+
+if not choice.isnumeric() or not 1 <= int(choice) <= 2:
+    print("Некорректно введены данные, попробуйте снова")
+    exit()
+
+if int(choice) == 1:
+    time = f"{datetime.datetime.now().hour}:{datetime.datetime.now().minute}"
+    if len(time.split(':')[1]) == 1:
+        time = time.split(':')[0] + ":0" + time.split(':')[1]
+    elif len(time.split(':')[0]) == 1:
+        time = "0" + time.split(':')[0] + ":" + time.split(':')[1]
```

Comment:
there is a better way to get this string like strftime, but you actually need separated values of hours and minutes, so might as well just assign them to new variables